src/Controls/StreetControls.js

```diff
@@ -178,26 +193,33 @@ class StreetControls extends FirstPersonControls {
             // state mouse drag (move + mouse click)
             this._stateOnMouseDrag = true;
             this.stopAnimations();
+        } else if (!this.tween) {
```

Comment:
I don't understand why you are moving it like this ?

Reply:
Because I would like stop picking when there is a animation

Reply:
Fair enough